docs/book/src/migration/multi-group.md

```diff
@@ -29,36 +28,29 @@ can check `api/v1/groupversion_info.go` to find that out:
 package v1
 ```
 
-Then, we'll rename `api` to `apis` to be more clear, and we'll move our
-existing APIs into a new subdirectory, "batch":
-
-```bash
-mv api apis
-mkdir apis/batch
-mv apis/v* batch/
-```
-
-Next, we'll need to update all the references to the old package name. For
-CronJob, that'll be `main.go` and `controllers/cronjob_controller.go`.  If
-you've added additional files to your project, you'll need to track down
-imports there as well.
-
-Finally, we'll add a new line to `PROJECT` that marks this as
-a multi-group project:
-
+When the command `kubebuilder multigroup` be executed it will add a new line 
+to `PROJECT` that marks this a multi-group project:
```

Comment:
This is a guide:

> Then we will notify kubebuilder that this is a multigroup project by executing:
> ```bash
> kubebuilder multigroup
> ```
> 
> This will modify the `Project` file to mark it as a multi-group project: